### DISCUSSION TITLE
Explorer Patcher works in Windows 11 build 26100 but has some issues

### DISCUSSION THREAD

**Discussion by btdzz** [valinet/ExplorerPatcher#3434] [General]
Explorer Patcher initially fails to install in Windows 11 build 26100 with this error: 
![image](https://github.com/valinet/ExplorerPatcher/assets/173114623/2c213f1b-f846-4e39-b40a-1298e9f83963)
Then I took an older version of Explorer Patcher and installed it, then updated to the latest version and it worked fine. To get the Windows 10 taskbar, I replaced the Windows 11 explorer.exe with the Windows 10 one. Everything works fine. One problem is that I can't uninstall Explorer Patcher. But this doesn't affect me too much because I'm about to reinstall Windows.
![image](https://github.com/valinet/ExplorerPatcher/assets/173114623/c3a50159-ad52-4983-95a0-14e27ce07ca6)

**Comment by Amrsatrio** [comment]
I have plans to rename some stuff to get around that PCA block because Microsoft hasn't been listening to us.

**Comment by Anixx** [comment]
So, one can restore the classic taskbar with replacing explorer.exe. I hope, Explorer Patcher would allow for this workaround even on later builds.

**Reply by Amrsatrio** [reply]
DM me on Discord. You are missing out on a lot of stuff I've done. I've pinged you in the Windhawk server.

**Comment by Anixx** [comment]
What version of Windows 10 Explorer did you use?
